.travis.yml

```diff
@@ -1,7 +1,9 @@
 sudo: false
 language: node_js
 script:
-  - npm test && npm run coveralls
+  - npm test
+  - npm run coveralls
```

Comment:
From Travis docs:
> When one of the build commands returns a non-zero exit code, the Travis CI build runs the subsequent commands as well, and accumulates the build result.

http://docs.travis-ci.com/user/customizing-the-build/#Customizing-the-Build-Step

So if `npm test` will fail Travis will try to execute `npm run coveralls` and it's produce more errors which will make error diagnostic harder.